codecov.yml

```diff
@@ -1,12 +1,18 @@
 # Don't post a comment on pull requests.
-comment: off
+comment: false
```

Comment:
It's ironic that YAML is written to be easy-to-write and according to http://yaml.org/type/bool.html, any possible negative string like "no", "off", and "false" are equivalent to `false`, but then some parsers are specific—that said, if that's what they need, that's fine. The config examples appear to use `false` so this change makes sense.

Reply:
They have a validator too (`curl --data-binary @codecov.yml https://codecov.io/validate`) but it might only be checking for general YAML syntax because all variants I tried validated fine.